docs/src/main/docs/guides/05_Dockerfile.adoc

```diff
@@ -0,0 +1,302 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+:adoc-dir: {guides-dir}
+
+= Creating Docker Images 
+:description: Helidon guide docker
+:keywords: helidon, guide, docker
+
+Building Docker images for Helidon applications.
+
+=== What You Will Learn
+
+You'll learn how to package your Helidon application and its runtime dependencies into a Docker
+image that also contains the Java runtime.
+
+=== What You Need
+
+|===
+|About 10 minutes
+| <<getting-started/01_prerequisites.adoc,Helidon Prerequisites>>
+|You'll also need Java 11 if you want to create custom JRE's using `jlink`
+|===
+
+=== Create Your Application Using The Helidon Quickstart
+
+Follow the instructions on the <<getting-started/02_base-example.adoc,Quickstart page>>
+to create a Helidon SE example. Once you've run the archetype to create
+the project come back here.
+
+=== Handling Runtime Dependencies
+
+Build the example:
+
+[source,bash,subs="verbatim,attributes"]
+----
+mvn clean package
+----
+
+When you run the maven build, you'll notice lines like the following:
+
+[source,bash,subs="verbatim,attributes"]
```

Comment:
In the recent review meeting of the RESTful web app guide we said we would use something other than `source` to display output to suppress the automatic "copy-to-clipboard" support. I've done a few very quick experiments and 
```
[listing]
----
example output here
----
```
should work nicely. the `[listing]` block also supports callouts.

Reply:
Nice! I'll give that a try.